### DISCUSSION TITLE
Commit history for a repository

### DISCUSSION THREAD

**Discussion by LeighWardle** [community/community#72895] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi all,

Using GitHub, how do I show the commit history for a repository?

I can access the history by using a URL like https://github.com/rust-lang/rust/commits/master

but I would prefer to find the commits via menus.

Regards,

Leigh

**Comment by Piero24** [comment, marked as answer]
@LeighWardle You can find it by clicking on the little clock with the number 237,492 (that are the number of commits) under the green code button on top of the repository. In your case: [rust](https://github.com/rust-lang/rust)

**Comment by LeighWardle** [comment]
Thanks, Piero.
Regards,
Leigh
